yarn-project/aztec/src/cli/aztec_start_options.ts

```diff
@@ -75,6 +75,12 @@ export const aztecStartOptions: { [key: string]: AztecStartOption[] } = {
       envVar: 'ENABLE_GAS',
       ...booleanConfigHelper(),
     },
+    {
```

Comment:
is this option ever useful for external decs? if so - maybe document?

Reply:
It will be, but right now not very much. Will do when PXE in the browser proves and is fully fleshed out!